pp/integrations/plugin-packs/procedures/network-juniper-isg-snmp.md

```diff
@@ -2,48 +2,524 @@
+
+> The services listed above are created automatically when the **Net-Juniper-Isg-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias  | Service Template                           | Service Description                          | Discovery  |
+|:---------------|:-------------------------------------------|:---------------------------------------------|:----------:|
+| Traffic-Global | Net-Juniper-Isg-Traffic-Global-SNMP-custom | Check traffic of multiple network interfaces | X          |
+
+> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.
+
+> If **Discovery** is checked, it means a service discovery rule exists for this service template.
+
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Host discovery
+
+| Rule name       | Description                                                                                                                                                                                                                                       |
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-Isg-SNMP-custom** host template |
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                         | Description                                                   |
+|:----------------------------------|:--------------------------------------------------------------|
+| Net-Juniper-Isg-SNMP-Traffic-Name | Discover network interfaces and monitor bandwidth utilization |
+
+More information about discovering services automatically is available on the [dedicated page](/docs/monitoring/discovery/services-discovery)
+and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Cpu" label="Cpu">
+
+| Metric name | Unit  |
+|:------------|:------|
+| cpu_1min    | %     |
+| cpu_5min    | %     |
+| cpu_15min   | %     |
+
+</TabItem>
+<TabItem value="Hardware-Global" label="Hardware-Global">
+
+Coming soon
```

Comment:
No metrics? 
If none, Explain that there are no metrics for this service template.